etc/kayobe/controllers.yml

```diff
@@ -14,9 +15,21 @@
 
 # List of default networks to which controller nodes are attached.
 #controller_default_network_interfaces:
+# Modify the default list of networks to remove storage management.
+controller_default_network_interfaces: >
+  {{ [provision_oc_net_name,
+      oob_wl_net_name,
+      provision_wl_net_name,
+      internal_net_name,
+      storage_net_name] | unique | list }}
```

Comment:
Why the list here? According to http://jinja.pocoo.org/docs/2.10/templates/#unique a list is returned.

Reply:
Some filters return generators, which need to be passed through the list filter. I'm not sure about unique, but this is how it's done in kayobe group_vars/all/controllers